sdcm/utils/common.py

```diff
@@ -1526,14 +1528,64 @@ def get_branched_ami(scylla_version: str, region_name: str, arch: AwsArchType =
     images = sorted(itertools.chain.from_iterable(images), key=lambda x: x.creation_date, reverse=True)
     images = [image for image in images if not image.name.startswith('debug-image')]
 
-    assert images, f"AMIs for {scylla_version=} with {arch} architecture not found in {region_name}"
+    if not images:
```

Comment:
again don't demote the assert into a log print, this function is being called also from other places which are no via the commandline, we do want it to fail.